src/plugins/timelion/public/plugin.ts

```diff
@@ -93,7 +103,8 @@ export class TimelionPlugin implements Plugin<void, void> {
       defaultPath: '#/',
       euiIconType: 'logoKibana',
       category: DEFAULT_APP_CATEGORIES.kibana,
-      updater$: this.appStateUpdater.asObservable(),
+      navLinkStatus:
+        visTypeTimelion.isUiEnabled === false ? AppNavLinkStatus.hidden : AppNavLinkStatus.default,
```

Comment:
This is addressing the issue and is fine to solve the GS bug, However just FYI:

Not sure if you only want to hide the navlink or to totally disable the app when `visTypeTimelion.isUiEnabled === false`. Hiding the navlink means that a user can still access the app via a link or by manually typing the url in the addressbar.

If, instead, you want to totally disable the app in that scenario, you can use `status: inaccessible` instead of `navLinkStatus: hidden`

Reply:
Thank you @pgayvallet for pointing this. Tbh, I didn't want to change the current behavior this is the reason I decided to only hide it. @timroes what do you think? We could do that, as we want to deprecate the Timelion app on v8 or do you think it will trouble the users?

Reply:
Hi, I've discussed that with Joe earlier. We both don't remember the exact reasoning but both assume it might be linked to that the vis and the application have been one plugin that time. We also think it might make more sense to just leave it like that, because we anyway remove it in 8.0 completely and we know that solution works, and maybe not change the mechanism completely right now. The `status: inaccessible` sounds like a solution we could nevertheless go with, since it should not change too much.